docs/jargon-mesh.md

```diff
@@ -0,0 +1,658 @@
+---
+id: terminology
+title: Temporal's core terminology
+sidebar_label: Terminology
+---
+
+import Mermaid from '@theme/Mermaid';
+
+## Workflow
+
+A fault-oblivious stateful function that orchestrates Activities.
```

Comment:
This up-front is to me confusing as:
1) Workflow execution **can** fail
2) User can write Workflow that invokes 0 Activities

Reply:
Not remotely ready for feedback.